src/astro/sql/__init__.py

```diff
@@ -63,6 +64,10 @@ def transform_file(
     warehouse: Optional[str] = None,
     output_table: Table = None,
 ):
+    warnings.warn(
+        "astro.sql.transform_file is now deprecated and will be taken out in version 1.0. Please use astro.sql.render instead"
+    )
```

Comment:
@dimberman since we are in an alpha version, and we have a disclaimer saying that the interface can change, wouldn't it make sense to just remove this feature now - as opposed to wait a stable release? This seems a less disruptive change than changing how we do query templating - and for that change we did not wait release `1.0.x`

Reply:
@tatiana fair enough, I'll just remove it